artemis/reporting/modules/ssh_bruter/translations/pl_PL/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,27 @@
+#: artemis/reporting/modules/ssh_bruter/template_exposed_ssh_with_easy_password.jinja2:2
+msgid ""
+"The following address contains SSH server where simple login/password "
+"pair allows logging in:"
+msgstr ""
+"Na następującym adresie uruchomiony jest serwer SSH przez który można się"
+" zalogować prostym hasłem:"
+
+#: artemis/reporting/modules/ssh_bruter/template_exposed_ssh_with_easy_password.jinja2:6
+msgid "the following credentials allow logging in"
+msgstr "Poniższe dane umożliwiają zalogowanie się"
+
+#: artemis/reporting/modules/ssh_bruter/template_exposed_ssh_with_easy_password.jinja2:10
+msgid "username"
+msgstr "użytkownikowi"
```

Comment:
jak piszesz "Poniższe dane umożliwiają zalogowanie się" to potem w zdaniu przypadek musi się zgadzać: "nazwą użytkwonika ... i hasłem"